apps/teamsjs-diagnostic-app/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "teamsjsdiagnosticapp",
```

Comment:
nit: I find `teamsjsdiagnosticapp` hard to read can we change this to `teamsjs-diagnostic-app` or `teams-diagnostic-app`